src/matchers/toBeChecked/index.ts

```diff
@@ -10,7 +10,7 @@ const toBeChecked: jest.CustomMatcher = async function (
 
     return {
       pass: isChecked,
-      message: () => getMessage(this, "toBeChecked", true, isChecked),
+      message: () => getMessage(this, "toBeChecked", true, isChecked, ""),
```

Comment:
This updates the matcher hint to more accurately reflect the matcher.  Instead of printing:

```js
expect(expected).toBeChecked(received)
```

it will print

```js
expect(expected).toBeChecked()
```